test/e2e/serial/tests/resource_hostlevel.go

```diff
@@ -108,6 +108,32 @@ var _ = Describe("[serial][hostlevel] numaresources host-level resources", Seria
 				schedOK, err := nrosched.CheckPODWasScheduledWith(fxt.K8sClient, updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
 				Expect(err).ToNot(HaveOccurred())
 				Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
+
+				By("wait for NRT data to settle")
+				e2efixture.MustSettleNRT(fxt)
+
+				targetNrtInitial, err := e2enrt.FindFromList(nrtList.Items, updatedPod.Spec.NodeName)
+				Expect(err).NotTo(HaveOccurred())
+
+				accumulatedRes := corev1.ResourceList{}
+				if expectedQOS == corev1.PodQOSGuaranteed {
+					noNrtFilter := func(resName corev1.ResourceName, resQty resource.Quantity) bool {
+						hostLevel := resName == corev1.ResourceEphemeralStorage
+						excCpus := true
+						if resName == corev1.ResourceCPU {
+							_, err := strconv.Atoi(resQty.String())
+							if err != nil && resQty.Value()%1000 > 0 {
+								excCpus = true
+							}
+						}
+						return !(hostLevel || excCpus)
+
```

Comment:
IIUC the intent:
```
if resName == corev1.ResourceEphemeralStorage {
    return false
}
if resName == corev1.ResourceCPU {
    cpuAmount, ok := cpuQty.AsInt64()
    if !ok {
        return false
    }
    if cpuAmount % 1000 > 0 {
        return false
    }
}
return true
```

Reply:
Feel free to borrow code (and point out mistakes, if any) from https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/pkg/noderesourcetopology/resourcerequests/exclusive.go

Reply:
yes that's the intention, I'll fix the confusion